### DISCUSSION TITLE
Transaction

### DISCUSSION THREAD

**Discussion by Peter254-prog** [blockscout/blockscout#10952] [General]
I sent worldcoin to my binance like 8 hours ago and I am yet to receive the coins in my binance wallet and the coins have already left my worldcoin account. What is the issue?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
